torch/csrc/jit/passes/graph_rewrite_helper.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/api/module.h>
```

Comment:
what is this import?

Reply:
It is need for `script::Module`. Recent code org change, I think took it out of ir.h. Compilation does not fail, I think, because def gets included from somewhere else.

Reply:
Well it seems, there is no script::Module in this header file. Thanks for the catch. I will remove it.